internal/build/docker.go

```diff
@@ -247,7 +250,23 @@ func (l *localDockerBuilder) startContainer(ctx context.Context, imgRef string,
 		if err != nil {
 			return "", err
 		}
-	case <-statusCh:
+	case status := <-statusCh:
+		if status.Error != nil {
+			return "", errors.New(status.Error.Message)
+		}
+		// TODO(matt) feed this reader into the logger
+		r, err := l.dcli.ContainerLogs(ctx, containerID, types.ContainerLogsOptions{ShowStdout: true, ShowStderr: true})
```

Comment:
ah, whelp, i believe we already had a func implementing some of this? `startContainerWithOutput` -- so whenever you actually do something with the output, you should remember to update that func too to remove any redundancies.